src/communication/HISTORY.rst

```diff
@@ -3,10 +3,17 @@
 Release History
 ===============
 
+2.0.0
```

Comment:
I'm assuming that the addition of MI commands will also be under 2.0.0?

Reply:
Also, is the intention to have this migration only for 2.0.0+? Minor version fixes (1.6.1 or 1.7.0) won't need to migrate to AAZ dev tools?

Reply:
I was thinking of using 2.1 for MI commands.
I think I didn't get the second comment.

Reply:
Re: 2nd comment, if we merge this change as part of a version bump to 2.0.0, that means that if we also want older versions to transition away from Autorest, that would make updating them more difficult as this is specifically part of 2.0.0 release onwards

Reply:
For the version, you can reference this [doc](https://microsoft.sharepoint.com/:w:/t/AzureCLITools237/EWG5EuACKmtNum6A3pyLakMBSaQnkjomC9XPz9ogNPS69A?e=d4NfTG) which will be released soon. If you have any questions, you can ask in the channel.